package.json

```diff
@@ -61,6 +62,7 @@
     "@material-ui/core": "^4.9.12",
     "@material-ui/icons": "^4.11.2",
     "@material-ui/lab": "^4.0.0-alpha.54",
+    "@material-ui/utils": "^5.0.0-beta.4",
```

Comment:
I don't know why but without adding it here, the build fails. The dependency is already added to the DataGrid's package.json.